deploy/crds/infinispan.org_infinispans_crd.yaml

```diff
@@ -810,6 +810,25 @@ spec:
                 - type
                 type: object
               type: array
+            podStatus:
+              properties:
+                ready:
+                  description: Deployments are ready to serve requests
```

Comment:
```suggestion
                  description: Deployments are ready to handle client requests
```

Reply:
@oraNod this is auto-generated values from the RedHat syseng operator-utils component. We can't to update values in our code